src/Chapter3.hs

```diff
@@ -910,6 +985,22 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = Gold Int
+
+instance Append Gold where
+  append :: Gold -> Gold -> Gold
+  append (Gold a)(Gold b) = Gold $ a + b
```

Comment:
It's usually a good idea to separate patterns by space :arrow_left: :space_invader: :arrow_right: 
```suggestion
  append (Gold a) (Gold b) = Gold $ a + b
```